adapter/cmd/adapter/conf/resources/adapter/security/keystore/mg.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
please generate new certs for the APK without copying from the old cc

Reply:
Created this issue: https://github.com/wso2/apk/issues/146